cmd/flux/image_test.go

```diff
@@ -53,6 +53,18 @@ func TestImageScanning(t *testing.T) {
 			"get image policy podinfo-regex",
 			"testdata/image/get_image_policy_regex.golden",
 		},
+		{
+			"suspend image policy podinfo-semver",
+			"testdata/image/suspend_image_policy.golden",
+		},
+		{
+			"resume image policy podinfo-semver",
+			"testdata/image/resume_image_policy.golden",
+		},
+		{
+			"reconcile image policy podinfo-semver",
+			"testdata/image/reconcile_image_policy.golden",
+		},
```

Comment:
Is this really enough for integration test? I am afraid running `make test` is not working completely on my env.  Bunch of `covdata` related issue and `resume image policy` doesn't seem to be matching `Use:   "policy [name]"`